ui/raidboss/data/03-hw/dungeon/sohm_al.txt

```diff
@@ -0,0 +1,141 @@
+
+# SOHM AL
+
+hideall "--sync--"
+hideall "--Reset--"
+
+0 "--Reset--" sync /00:0839:.*is no longer sealed/ window 100000 jump 0
+
+### Raskovnik
+
+0 "Start" sync /00:0839:Greenlinn will be sealed off/ window 0,1
+6.2 "Bloody Caress" sync /:Raskovnik:ED1:/ window 6.2,0
+12.3 "--sync--" sync /:Raskovnik:ED2:/
+17.9 "Acid Rain" sync /:Raskovnik:ED7:/
+26.4 "Phytobeam" sync /:Raskovnik:ED5:/
+33.6 "Bloody Caress" sync /:Raskovnik:ED1:/ jump 6.2
+39.7 "--sync--" # sync /:Raskovnik:ED2:/
+45.3 "Acid Rain" # sync /:Raskovnik:ED7:/
+53.9 "Phytobeam" # sync /:Raskovnik:ED5:/
+61.1 "Bloody Caress" # sync /:Raskovnik:ED1:/
+
+# HP push?
+# Looks like there are two loop groups of skills used repeatly?
+
+# Group 1
+100.0 "--sync--" sync /:1395:Raskovnik/ window 100,0
+103.0 "Sweet Scent" sync /:Raskovnik:1395:/ window 103,0
+114.6 "Floral Trap" sync /:Raskovnik:1391:/
+119.6 "Flower Devour" sync /:Raskovnik:1392:/
+125.4 "Spit" sync /:Raskovnik:1393:/
+126.5 "--sync--" sync /:Raskovnik:1394:/
+136.0 "Floral Trap" sync /:Raskovnik:1391:/ jump 114.6
+141.0 "Flower Devour" # sync /:Raskovnik:1392:/
+146.8 "Spit" # sync /:Raskovnik:1393:/
+147.9 "--sync--" # sync /:Raskovnik:1394:/
+157.1 "Floral Trap" # sync /:Raskovnik:1391:/
+162.1 "Flower Devour" # sync /:Raskovnik:1392:/
```

Comment:
Why are all of these commented out?

Reply:
Oh, yes yes.
I was going to define two loops but forget to add a window keyword.
Here is a loop inside a loop, the outside is from "Sweet Scent", and the inside is "Trap" to "Spit".
I think I should add one more `1A StartsCasting` line to match "Sweet Scent"?

Reply:
> Why are all of these commented out?

Because it is after a jump?